Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/it/GroupControllerIT.java

```diff
@@ -25,60 +31,66 @@
 /**
  * @author - wally55077@gmail.com
  */
+@Transactional
 @ActiveProfiles(Profiles.JWT)
 @ContextConfiguration(classes = SpringBootStudentApplication.class)
 public class GroupControllerIT extends AbstractSpringBootTest {
 
-    private static final String TEST_NAME = "name";
-    private static final String BASE_PATH = "/api/groups";
+    private static final String GROUP_NAME = "groupName";
```

Comment:
I missed this one.


In this test class, you are having typos in `Give`s which should be `Given`s.
Please correct all of them.